app/api/images/images.controller.js

```diff
@@ -587,8 +595,9 @@ exports.updateAttributes = utils.route(async (req, res) => {
   }
 
   //check iiif_data
-  if (req.body.iiif_data.regionByPx) {
-    const [x,y,w,h] = req.body.iiif_data.regionByPx;
+  const regionByPx = req.body.iiif_data ? req.body.iiif_data.regionByPx : undefined;
+  if (regionByPx) {
+    const [x,y,w,h] = regionByPx;
```

Comment:
Have you tested the values here as i think this will set the first characters in the string like
regionByPx ="one,two,three,four";
x=o
y=n
w=e
h=,

This maybe should be changed to const [x,y,w,h] = regionByPx.split(',');

Reply:
should work if clearly defined as integers inside the JSON

Reply:
The JSON schema only accepts array of integers, so yes this should always work.